contrib/beginner_example/inputs.json

```diff
@@ -1,3 +1,3 @@
 {
-  "bam_chrom_counter.bam" : "dx://project-BfZj3KQ03zx77KF7pjjBFZkf:/testexomes/testexome.pair0.tumour.bam"
+  "bam_chrom_counter.bam" : "dx://dxCompiler_playground:/test_data/wgEncodeUwRepliSeqBg02esG1bAlnRep1.bam"
```

Comment:
is this a staging project?

Reply:
I updated the input to a file hosted on PROD in project "DNAnexus Regression Testing Project AWS US east"